File: IncAng/src/dto/CodeDTO.ts
import { BehaviorDTO } from './BehaviorDTO';

export class CodeDTO{
    idCode: number;
    name: String;
    body: String;
    type: String;
    behaviorDTO: BehaviorDTO;

constructor(id: number, name: String, body: String, type: String, idBehavior: number){
    this.idCode = id;
    this.name = name;
    this.body = body;
    this.type = type;
    idBehavior = this.behaviorDTO.idBehavior;
}
}

File: RobotMicroservice/src/main/java/it/contrader/dto/CodeDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class CodeDTO {
		private Integer idCode;
		private String name;
		private String body;
		private String type;
		private BehaviorDTO behavior = new BehaviorDTO();
}
